sdk/src/reader.rs

```diff
@@ -136,7 +136,13 @@ impl Reader {
             Store::from_stream_async(format, &mut stream, verify, &mut validation_log).await
         }?;
 
-        Self::from_store(store, &validation_log)
+        let /* mut */ result = Self::from_store(store, &validation_log)?;
+        if false {
+            // QUESTION: What to do if we're in the _sync version and there
```

Comment:
Good question. I wouldn't want everything to become async. I wonder if we need the sync interface for some native bindings? It's not great but maybe we need a blocking API? Or a cawg validation opt-out for the sync path?